reference/promise-types/edit_line.markdown

```diff
@@ -410,8 +411,37 @@ mark out the region of a file to be edited.
      select_end => "\[.*\]";
      }
 ```
-
 If you want to match from a starting location to the end of the file
 (even if there are other lines matching `select_start` intervening),
 then just omit the `select_end` promise and the selected region will run
 to the end of the file.
+The default behavior can be modified by using  [`select_end_match_eof`][bundle edit_line#select_end_match_eof]. 
+
+#### select_end_match_eof
+
+**Description:** Whether to match end of file as the end on the region if `select_end` was not matched for the given region.
+If `select_end_match_eof` is true, the selected region will run to the end of the file. 
```

Comment:
I think I would simplify the wording a bit.

When `select_end_match_eof` is set to true `select_end` will consider end of file as the end region if it is unable to match.